webservices/common/models/elections.py

```diff
@@ -27,3 +27,14 @@ class ElectionsList(db.Model):
     cycle = db.Column(db.Integer)
     incumbent_id = db.Column(db.String, doc=docs.CANDIDATE_ID)
     incumbent_name = db.Column(db.String, doc=docs.CANDIDATE_NAME)
+
+
+class ZipsDistricts(db.Model):
+    __table_args__ = {'schema': 'staging'}
```

Comment:
The `staging` schema?  I may have missed this, so I just wanted to double check on it to make sure I know what it is. :-)

Reply:
@ccostino - it looks like that's where the table was put, with [this migration](https://github.com/18F/openFEC/blob/develop/data/migrations/V0050__add_ref_zip_to_district.sql). Is that ok?

Reply:
that was already added here: https://github.com/18F/openFEC/blob/83f59d5c2c1d4aa90aac036b276d4ca54340dbaa/data/migrations/V0050__add_ref_zip_to_district.sql